backend/src/mastodon/status.ts

```diff
@@ -2,13 +2,16 @@ import type { Handle } from '../utils/parse'
 import type { MediaAttachment } from 'wildebeest/backend/src/types/media'
 import type { UUID } from 'wildebeest/backend/src/types'
 import { getObjectByMastodonId } from 'wildebeest/backend/src/activitypub/objects'
-import type { Note } from 'wildebeest/backend/src/activitypub/objects/note'
+import { createPublicNote, type Note } from 'wildebeest/backend/src/activitypub/objects/note'
 import { loadExternalMastodonAccount } from 'wildebeest/backend/src/mastodon/account'
 import * as actors from 'wildebeest/backend/src/activitypub/actors'
 import * as media from 'wildebeest/backend/src/media/'
 import type { MastodonStatus } from 'wildebeest/backend/src/types'
 import { parseHandle } from '../utils/parse'
 import { urlToHandle } from '../utils/handle'
+import type { Person } from 'wildebeest/backend/src/activitypub/actors'
+import { addObjectInOutbox } from '../activitypub/actors/outbox'
+import type { Object as ActivityPubObject } from 'wildebeest/backend/src/activitypub/objects'
```

Comment:
In a future PR we should just rename `Object` to `APObject`.

Reply:
yeah 100% agree, `Object` is extremely ambiguous and also the vscode fails to auto-import it (since it doesn't see a missing type)

I would likely go with `ActivityPubObject` I think, but I tend to be rather verbose with my naming 😁 😬